saleor/settings.py

```diff
@@ -250,7 +250,9 @@ def get_bool_from_env(name, default_value):
     'phonenumber_field',
     'captcha']
 
-if DEBUG:
+
+ENABLE_DEBUG_TOOLBAR = get_bool_from_env('ENABLE_DEBUG_TOOLBAR', False)
```

Comment:
Should we really disable it by default?

Reply:
If we don't want the next line
`if DEBUG and ENABLE_DEBUG_TOOLBAR:` then yes. Otherwise, it will be added on production servers by default and we for sure don't want that.